hack/test-e2e.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+#shellcheck disable=SC2086
+
+set -e
+
+image="origin-libvirt-machine-controllers:latest"
+script_dir="$(cd $(dirname "${BASH_SOURCE[0]}") && pwd -P)"
+
+# Set packet instance connection parameters
+RHOST="root@$(cat /tmp/packet_ip)"
+SSH_OPTS="-o UserKnownHostsFile=/dev/null -o IdentitiesOnly=yes -o PasswordAuthentication=no -o StrictHostKeyChecking=no -i /tmp/packet_id_rsa"
+
+# TODO remove after https://github.com/paulfantom/packet-image/pull/1
+ssh ${SSH_OPTS} "${RHOST}" 'yum -y -d1 install docker && systemctl start docker'
+
+# Import fedora-28 image into libvirt
+ssh ${SSH_OPTS} "${RHOST}" 'virsh vol-create-as default fedora_base $(stat -Lc%s /Fedora-Cloud-Base-28-1.1.x86_64.qcow2) --format raw'
+ssh ${SSH_OPTS} "${RHOST}" 'virsh vol-upload --pool default fedora_base /Fedora-Cloud-Base-28-1.1.x86_64.qcow2'
+
+# Copy ssh keys to packet host
+scp ${SSH_OPTS} ${script_dir}/../cmd/libvirt-actuator/resources/guest.pem "${RHOST}":/guest.pem
```

Comment:
We should move the key under the `hack` directory and copy it from there. `cmd/libvirt-actuator/resources` is more like a temporary holder for all pieces needed to bootstrap a cluster, resp. run create/exists/delete ops.